<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAbout</class>
 <widget class="QDialog" name="CAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1" colspan="2">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="btnGitHub">
        <property name="text">
         <string>GitHub</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="labVersion">
       <property name="text">
        <string>version</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="labInfo">
       <property name="text">
        <string>内容</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="labPic">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>CAbout</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
